src/features/map/components/Map/Layers/ConstructionStandardLegend.jsx

```
@@ -65,14 +57,10 @@ const ConstructionStandardLegend = () => {
       >
         {title}
-        <Tooltip
-          title={
-            isConstruction ? INFO_TEXT.construction : INFO_TEXT.mainUseType
-          }
-          styles={{ body: { fontSize: 12 } }}
-        >
-          <InfoCircleOutlined style={{ color: '#999', cursor: 'pointer' }} />
-        </Tooltip>
+        <InfoTooltip
+          tooltipKey={isConstruction ? 'construction-archetypes-info' : 'main-use-type-info'}
```

ESLint

[error] Replace `isConstruction·?·'construction-archetypes-info'·:·'main-use-type-info'` with `⏎············isConstruction⏎··············?·'construction-archetypes-info'⏎··············:·'main-use-type-info'⏎··········`

(prettier/prettier)